src/gt4py/next/ffront/decorator.py

```diff
@@ -705,13 +706,33 @@ def __call__(
                 )
```

Comment:
Should field_operator as program return the `out` param? Maybe yes...

Reply:
I would say no as programs don't return anything.

Reply:
than it was correct, right?

Reply:
No, the program does not return something, but the field operator which we are calling does.

Reply:
no, we `return self.as_program()()` i.e. whatever program does which is currently nothing therefore still correct and in case (for whatever reason) `program()` returns something, we will do the right thing implicitly.